Code/Framework/AzFramework/AzFramework/DocumentPropertyEditor/ReflectionAdapter.cpp

```diff
@@ -566,9 +566,9 @@ namespace AZ::DocumentPropertyEditor
                 parentContainerInstanceAttribute && !parentContainerInstanceAttribute->IsNull())
             {
                 auto containerEntry = m_containers.ValueAtPath(m_builder.GetCurrentPath(), AZ::Dom::PrefixTreeMatch::ExactPath);
-                if (containerEntry)
+                if (containerEntry && containerEntry->m_element)
```

Comment:
This actually causes a bug. Removing it.

Reply:
Yeah, the `m_element` should only be set if is nullptr.

If it is not nullptr then it has already been reserved and is in the container.